test/size.txt

```diff
@@ -1,53 +1,55 @@
-103.49 kB   |     4.91 kB   |    2.15 kB   |   v0/amp-dynamic-css-classes-0.1.js
-120.66 kB   |     8.87 kB   |    3.63 kB   |   v0/amp-experiment-0.1.js
-164.97 kB   |     15.4 kB   |    6.29 kB   |   v0/amp-facebook-0.1.js
- 36.68 kB   |     5.84 kB   |    2.46 kB   |   v0/amp-fit-text-0.1.js
-110.46 kB   |     7.79 kB   |    3.17 kB   |   v0/amp-font-0.1.js
-152.38 kB   |    12.64 kB   |    5.01 kB   |   v0/amp-form-0.1.js
- 99.31 kB   |     8.02 kB   |     3.2 kB   |   v0/amp-fx-flying-carpet-0.1.js
-113.21 kB   |     8.34 kB   |    3.42 kB   |   v0/amp-google-vrview-image-0.1.js
-161.22 kB   |    14.75 kB   |    5.87 kB   |   v0/amp-iframe-0.1.js
-241.14 kB   |    31.29 kB   |   10.29 kB   |   v0/amp-image-lightbox-0.1.js
-113.03 kB   |      7.3 kB   |    3.04 kB   |   v0/amp-instagram-0.1.js
-  94.4 kB   |     8.08 kB   |    3.42 kB   |   v0/amp-install-serviceworker-0.1.js
- 93.35 kB   |     7.21 kB   |    2.99 kB   |   v0/amp-jwplayer-0.1.js
-126.07 kB   |      8.3 kB   |    3.38 kB   |   v0/amp-kaltura-player-0.1.js
-147.98 kB   |     11.8 kB   |    4.43 kB   |   v0/amp-lightbox-0.1.js
- 95.47 kB   |     5.68 kB   |    2.46 kB   |   v0/amp-list-0.1.js
-163.46 kB   |    13.19 kB   |    4.99 kB   |   v0/amp-live-list-0.1.js
-149.09 kB   |    39.76 kB   |   14.27 kB   |   v0/amp-mustache-0.1.js
- 86.56 kB   |      6.6 kB   |    2.77 kB   |   v0/amp-o2-player-0.1.js
-136.32 kB   |    20.05 kB   |       6 kB   |   v0/amp-pinterest-0.1.js
- 85.56 kB   |     6.09 kB   |    2.57 kB   |   v0/amp-reach-player-0.1.js
- 81.63 kB   |     5.57 kB   |    2.32 kB   |   v0/amp-share-tracking-0.1.js
- 98.87 kB   |     9.94 kB   |    3.75 kB   |   v0/amp-sidebar-0.1.js
- 117.9 kB   |    10.86 kB   |    4.27 kB   |   v0/amp-slides-0.1.js
-116.24 kB   |    10.96 kB   |    4.18 kB   |   v0/amp-social-share-0.1.js
- 86.41 kB   |     6.31 kB   |    2.65 kB   |   v0/amp-soundcloud-0.1.js
- 94.17 kB   |     7.91 kB   |    3.11 kB   |   v0/amp-springboard-player-0.1.js
-110.34 kB   |     8.04 kB   |    3.24 kB   |   v0/amp-sticky-ad-0.1.js
-165.39 kB   |    15.52 kB   |    6.33 kB   |   v0/amp-twitter-0.1.js
-123.17 kB   |    10.08 kB   |    3.93 kB   |   v0/amp-user-notification-0.1.js
- 86.06 kB   |     6.18 kB   |    2.61 kB   |   v0/amp-vimeo-0.1.js
- 85.59 kB   |     6.04 kB   |    2.57 kB   |   v0/amp-vine-0.1.js
-115.26 kB   |     7.26 kB   |    3.05 kB   |   v0/amp-viz-vega-0.1.js
-133.08 kB   |     8.87 kB   |    3.65 kB   |   v0/amp-youtube-0.1.js
-179.58 kB   |    39.34 kB   |   13.61 kB   |   current-min/f.js / current/integration.js
\ No newline at end of file
+      max   |         min   |        gzip   |   file
+      ---   |         ---   |         ---   |   ---
+  58.5 kB   |     5.42 kB   |     2.37 kB   |   alp.js / alp.max.js
+ 710.5 kB   |   140.48 kB   |    44.05 kB   |   shadow-v0.js / amp-shadow.js
+    787 B   |       406 B   |       sw.js
+ 83.97 kB   |    19.54 kB   |   sw.max.js
```

Comment:
can you add `normalizeRow(rows, 'sw.js', 'sw.max.js', true);`
here https://github.com/ampproject/amphtml/blob/master/build-system/tasks/size.js#L98

will fix this bug, currently it doesn't automatically relate these two files together since it isnt under extensions and our other top level files don't all follow the `.max` convention.